examples/seismic/acoustic/operators.py

```diff
@@ -126,13 +134,16 @@ def ForwardOperator(model, geometry, space_order=4,
         Defaults to False.
     kernel : str, optional
         Type of discretization, 'OT2' or 'OT4'.
+    factor : int, optional
+        Downsampling factor to save snapshots of the wavefield.
     """
     m = model.m
 
     # Create symbols for forward wavefield, source and receivers
     u = TimeFunction(name='u', grid=model.grid,
-                     save=geometry.nt if save else None,
+                     save=geometry.nt if save and factor is None else None,
```

Comment:
I'm not a big fan of this composite conditional involving both `save` and `factor`, which is also repeated across other modules

Reply:
Moved the conditional statement outside of the `u` definition but I do not know if there is a better way to avoid the composite conditional statement.